app/controllers/int_api/marketplaces_controller.rb

```diff
@@ -74,4 +76,10 @@ def set_access_control_headers
     headers['Access-Control-Allow-Origin'] = '*'
   end
 
+  def assign_onboarding_feature_flag(community_id:)
+    if(rand(2) == 1)
```

Comment:
This is nicely random. I had to test 5 times on local dev environment before I got a marketplace without feature flag.